src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -93,6 +93,8 @@ public override bool Execute()
                 Coverage coverage = null;
                 using (Stream instrumenterStateStream = fileSystem.NewFileStream(InstrumenterState.ItemSpec, FileMode.Open))
                 {
+                    var instrumentationHelper = DependencyInjection.Current.GetService<IInstrumentationHelper>();
```

Comment:
Please add a comment and link the msbuild repo issue.

Reply:
Done